common/cryptographer.py

```diff
@@ -375,3 +382,77 @@ def get_compressed_pk(pk):
 
         except TypeError as e:
             raise InvalidKey("PublicKey has invalid initializer", error=str(e))
+
+
+    @staticmethod
+    def generate_cert_key():
```

Comment:
Got it, sounds good, I'll change to ECDSA. The reason I used PEM is because that's the format GRPC accepts. Sound okay for me to save the key/certificate as DER, and then just convert to PEM when we need to use them in GRPC?

Reply:
Yeah, I guess you can actually create Key and Cert objects with `cryptography` and the just call the pem / serialize_pem / whatever method gets the pem (I cannot recall atm) when passing the data to `grpc.ssl_server_credentials`.

If it ends up being too complicated it won't be worth it, so feel free to weight it and decide whatever you see fits best.